### PR TITLE
 #166240938 Setup an empty react project

### PR DESCRIPTION
**Pull request by deschantkn**
#### What does this PR do?
This PR creates an empty react project.

#### Description of Task to be completed?
- Adding Webpack 4
- Adding Babel 7
- Adding eslint with the airbnb style guide

#### How should this be manually tested?
This task can be tested by cloning into this repo and running the following command:
`yarn dev`

#### Any background context you want to provide?
N/A

#### What are the relevant pivotal tracker stories?
 #166240938

#### Screenshots (if appropriate)
N/A

#### Questions:
N/A

### CODE REVIEW

**Review by mifeille — APPROVED** [review]

**Review by eliemugenzi — APPROVED** [review]
LGTA

**Review by Diama1 — APPROVED** [review]

**Comment by UhiriweAudace** [comment]
can you add also the Hound CI? as I saw above it seemed like you configured the eslint for this front-end repo

File: .eslintrc [hunk 1]
```diff
@@ -0,0 +1,7 @@
+{
+  "parser": "babel-eslint",
+  "extends": ["airbnb"],
+  "rules": {
+    "react/jsx-filename-extension": [1, { "extensions": [".js", ".jsx"] }]
```
Comment (UhiriweAudace) [inline .eslintrc:5]: I think need more rules, apart from this.

**Review by Gidraf — APPROVED** [review]
LGTM

**Pull request merged**